yarn-project/p2p/src/services/peer-manager/peer_manager.ts

```diff
@@ -45,7 +45,11 @@ export class PeerManager {
   private trustedPeersInitialized: boolean = false;
 
   private metrics: PeerManagerMetrics;
-  private discoveredPeerHandler;
+  private handlers: {
+    handleConnectedPeerEvent: (e: CustomEvent<PeerId>) => void;
+    handleDisconnectedPeerEvent: (e: CustomEvent<PeerId>) => void;
+    handleDiscoveredPeer: (enr: ENR) => Promise<void>;
+  } = {} as any;
```

Comment:
Why the `as any`? :-(

Reply:
Yeah this doesn't seem needed, we just want this to be | undefined and to use ! if typescript cant figure out where it is defined